docs/tutorial.ipynb

```diff
@@ -273,7 +273,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "IntegerType(parameters=[IntAttr(data=64)], width=IntAttr(data=64)) should be of base attribute int\n"
+      "In integer_type attribute verifier: 2 parameters expected, got 1\n"
```

Comment:
I am surprised. What triggered this change?

Reply:
I think this is related to #455

Reply:
I re-ran the cells, and this is what I got

Reply:
Ah, I see....